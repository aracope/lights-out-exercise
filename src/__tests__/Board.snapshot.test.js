
import React from "react";
import { render } from "@testing-library/react";
import Board from "../Board";

test("matches snapshot", () => {
  const { asFragment } = render(<Board nrows={2} ncols={2} chanceLightStartsOn={false} />);
  expect(asFragment()).toMatchSnapshot();
});
