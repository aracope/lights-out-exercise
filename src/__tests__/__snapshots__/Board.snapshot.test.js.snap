// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Board-title"
  >
    <h1>
      YOU WIN!
    </h1>
  </div>
</DocumentFragment>
`;
